### ISSUE TITLE
Address accessibility violations introduced by ember-phone-input

### ISSUE THREAD

**Issue by robustdj** [cardstack/ui-components#68]
When implementing #57 we introduced a few accessibility violations that we ended up ignoring since the designs are still in flux. We should come back and fix these violations by rendering a11y-friendly markup.

The config is stored in `helpers/axe-configuration.js`. Ideally we should be able to remove this config entirely
